04.ls/ls.rb

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env ruby
+# frozen_string_literal: true
+
+acquired_file = Dir.glob('*')
+NUMBER_OF_COLUMN = 3
```

Comment:
定数はファイル先頭に書くのが通例なので、こうしたほうが良いと思います。
```suggestion
NUMBER_OF_COLUMN = 3
acquired_file = Dir.glob('*')
```